geode-cq/src/distributedTest/java/org/apache/geode/internal/cache/tier/sockets/DurableClientTestBase.java

```diff
@@ -215,7 +215,7 @@ public void run2() throws CacheException {
                 .info(st + " CCP states: " + getAllClientProxyState());
             CacheServerTestUtil.getCache().getLogger().info(st + " CHM states: "
                 + printMap(
-                    ClientHealthMonitor._instance.getConnectedClients(null)));
+                    ClientHealthMonitor.instance.getConnectedClients(null)));
```

Comment:
how about changing "instance" to be private and have this code use the public "getInstance()"?